src/scheduler/MpiWorld.cpp

```diff
@@ -123,6 +123,12 @@ void MpiWorld::create(faabric::Message& call, int newId, int newSize)
             msg.set_cmdline(thisRankMsg->cmdline());
             msg.set_inputdata(thisRankMsg->inputdata());
             msg.set_migrationcheckperiod(thisRankMsg->migrationcheckperiod());
+            // To run migration experiments easily, we may want to propagate
+            // the `UNDERFULL` topology hint. In general however, we don't
+            // need to propagate this field
+            if (thisRankMsg->topologyhint() == "UNDERFULL") {
+                msg.set_topologyhint(thisRankMsg->topologyhint());
+            }
```

Comment:
It's not enforced in the style guide but after a stand-alone `if` I'd expect a linebreak.